sites/partners/src/applications/PaperApplicationDetails/DetailsMemberDrawer.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react"
+import {
+  AppearanceStyleType,
+  t,
+  GridSection,
+  ViewItem,
+  Button,
+  Drawer,
+} from "@bloom-housing/ui-components"
+import { AddressColsType, DetailsAddressColumns } from "./DetailsAddressColumns"
+import { Application, HouseholdMemberUpdate } from "@bloom-housing/backend-core/types"
+
+export type MembersDrawer = HouseholdMemberUpdate | null
+
+type DetailsMemberDrawerProps = {
+  application: Application
+  membersDrawer: MembersDrawer
+  setMembersDrawer: (member: MembersDrawer) => void
+}
+
+const DetailsMemberDrawer = ({
+  application,
+  membersDrawer,
+  setMembersDrawer,
+}: DetailsMemberDrawerProps) => {
+  return (
+    <Drawer
+      open={!!membersDrawer}
+      title={t("application.household.householdMember")}
+      ariaDescription={t("application.household.householdMember")}
+      onClose={() => setMembersDrawer(null)}
+    >
+      <section className="border rounded-md p-8 bg-white mb-8">
+        <GridSection
+          title={t("application.details.householdMemberDetails")}
+          tinted={true}
+          inset={true}
+          grid={false}
+        >
+          <GridSection grid columns={4}>
+            <ViewItem
+              label={t("application.name.firstName")}
+              children={membersDrawer?.firstName || t("t.n/a")}
+            />
+
+            <ViewItem
+              label={t("application.name.middleName")}
+              children={membersDrawer?.middleName || t("t.n/a")}
+            />
+
+            <ViewItem
+              label={t("application.name.lastName")}
+              children={membersDrawer?.lastName || t("t.n/a")}
+            />
+
+            <ViewItem
+              label={t("application.household.member.dateOfBirth")}
+              children={
+                membersDrawer?.birthMonth && membersDrawer?.birthDay && membersDrawer?.birthYear
+                  ? `${membersDrawer?.birthMonth}/${membersDrawer?.birthDay}/${membersDrawer?.birthYear}`
```

Comment:
Can this place use a date formatted with moment the same it is done here https://github.com/bloom-housing/bloom/pull/926/files#diff-5dfa3dfe3eac7e1e888dfba8e985573089e7ff584d3d9ad79b9c90c58300ff7aR26 ?

Reply:
As we discussed on Slack, it's not necessary in this case.